src/pkg/reg/adapter/jfrog/adapter.go

```diff
@@ -201,6 +209,19 @@ func (a *adapter) FetchArtifacts(filters []*model.Filter) ([]*model.Resource, er
 	return resources, nil
 }
 
+func (a *adapter) listAllRepositories() ([]*repository, error) {
+	var repositories []*repository
+	for _, t := range supportedTypes {
```

Comment:
Does JFrog not provide an interface to query all repositories? If using this approach of specifying a list, do scenarios such as future upgrades always require specifying via environment variables?